integrations/nlu-snips/src/SnipsNlu.ts

```diff
@@ -0,0 +1,197 @@
+import { join as joinPaths, resolve } from 'path';
+import { v4 as uuidV4 } from 'uuid';
+import {
+  axios,
+  AxiosRequestConfig,
+  AxiosResponse,
+  DynamicEntitiesMode,
+  Extensible,
+  HandleRequest,
+  Jovo,
+  JovoError,
+  JovoRequest,
+  NluData,
+  NluPlugin,
+  OutputTemplate,
+} from '@jovotech/framework';
+import { InputType, Intent, JovoModelData } from 'jovo-model';
+import { SnipsNluConfig, SnipsNluResponse } from './interfaces';
+
+export class SnipsNlu extends NluPlugin<SnipsNluConfig> {
```

Comment:
This whole class has to be updated to the beta version. The signature of `process` changed for example.

Reply:
Done :+1: